### PR TITLE
Upgrade tooling for WPCS 3.0 and PHP 8.0+

### PR DESCRIPTION
**Pull request by kadamwhite**
- Update to WPCS 3.0, and migrate to a temporary standard adapted from HM's outdated standard but adjusted for modern PHP compatibility
- Pin versions of remaining Action dependencies missed in #56 
- Introduce PHPCS testing actions

### CODE REVIEW

**Comment by tomjn** [comment]
I understand pinning to versions but why pin to a specific commit of the actions?

File: .github/workflows/php-standards.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    name: PHP Coding Standards
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Install PHP
+        uses: shivammathur/setup-php@e6f75134d35752277f093989e72e140eaa222f35 # v2.28.0
+        with:
+          php-version: '8.1'
```
Comment (tomjn) [inline .github/workflows/php-standards.yml:27]: do we want a matrix of versions here?

File: .travis.yml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # Travis CI Configuration File
```
Comment (tomjn) [inline .travis.yml:1]: do we still need Travis if we have a php linting action?

**Review by tomjn — APPROVED** [review]
Approved in principle from a reading of the code, left some comments

File: .travis.yml (hunk 2 above)
Comment (kadamwhite) [reply .travis.yml:1]: Unit tests! 😁

**Comment by jrfnl** [comment]
Thank you for your support and your enthousiasm embracing the take-over of the PHP_CodeSniffer package.

In contrast to earlier information, arrangements have been made to allow the package to continue under its original name on Packagist. The commit (in the new repo) to rename the package [has been reverted](https://github.com/PHPCSStandards/PHP_CodeSniffer/pull/135).

I'd recommend reverting the Composer reference changes and keeping the changes which refer to the repo URL on GitHub.

Sorry for the confusion and thank you for understanding. I hope you'll enjoy the [3.8.0 release](https://github.com/PHPCSStandards/PHP_CodeSniffer/releases/tag/3.8.0), which was released last month.

**Pull request merged**